childcare_center/remove_center_public_id_column.sql

```diff
@@ -0,0 +1,3 @@
+-- Removes the public id from the childcare center table. BIC_NUMBER isnt a proper identifier
+ALTER TABLE childcare_center DROP COLUMN public_id;
+ALTER TABLE childcare_center MODIFY COLUMN center_name VARCHAR(180) NOT NULL UNIQUE;
```

Comment:
Put this in another migration file since it's separate from removing the public_id column